target/linux/ramips/dts/mt7621_zte_mf286c3.dts

```diff
@@ -0,0 +1,350 @@
+			function-enumerator = <2>;
+			gpios = <&aw9523 1 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+
+		led-8 { /* p0_2 sig02 */
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_INDICATOR;
+			function-enumerator = <3>;
+			gpios = <&aw9523 2 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+
+		led-9 { /* p0_3 - sig03 */
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_INDICATOR;
+			function-enumerator = <4>;
+			gpios = <&aw9523 3 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+
+		led-3 { /* p0_5 - rgb05-red */
+			color = <LED_COLOR_ID_RED>;
+			function = LED_FUNCTION_INDICATOR;
+			function-enumerator = <5>;
+			gpios = <&aw9523 5 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+
+		led_status: led-0 { /* p0_6 - power_06 */
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_STATUS;
+			gpios = <&aw9523 6 GPIO_ACTIVE_LOW>;
+		};
+
+		led-4 { /* p1_0 - rgb10-blue */
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_INDICATOR;
+			function-enumerator = <6>;
+			gpios = <&aw9523 8 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+
+		led-10 { /* p1_1 - sig11 */
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_INDICATOR;
+			function-enumerator = <7>;
+			gpios = <&aw9523 9 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+
+		led-1 { /* p1_2 - wifi12 */
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_WLAN;
+			gpios = <&aw9523 10 GPIO_ACTIVE_LOW>;
+		};
+
+		led-5 { /* p1_3 - rgb13-green */
+			color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_INDICATOR;
+			function-enumerator = <8>;
+			gpios = <&aw9523 11 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+
+		led-2 { /* p1_4 - phone14 */
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_INDICATOR;
+			function-enumerator = <9>;
+			gpios = <&aw9523 12 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <50>;
+		pinctrl-names = "default";
+		pinctrl-0 = <&button_pins>;
+
+		key-wps {
+			gpios = <&aw9523 14 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+
+		key-wlan {
+			gpios = <&aw9523 15 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WLAN>;
+		};
+	};
+};
+
+&nand {
+	status = "okay";
+
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		partition@0 {
+			label = "Bootloader";
+			reg = <0x0 0x80000>;
+			read-only;
+		};
+
+		partition@80000 {
+			label = "Config";
+			reg = <0x80000 0x80000>;
+			read-only;
+		};
+
+		partition@100000 {
+			label = "Factory";
+			reg = <0x100000 0x100000>;
+			read-only;
+
+			nvmem-layout {
+				compatible = "fixed-layout";
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				eeprom0: eeprom@0 {
+					reg = <0x0 0x400>;
+				};
+
+				eeprom1: eeprom@8000 {
+					reg = <0x8000 0x200>;
+				};
+
+				macaddr_factory_4: macaddr@4 {
+					reg = <0x4 0x6>;
+				};
+
+				macaddr_factory_e000: macaddr@e000 {
```

Comment:
Please order those sections in the order of rising address, i.e, move macaddr@8004 here.